Automattic/simplenote-ios
# Revert Dropping collaboration messaging

This reverts commit 4c699914fd22e9af7d9ac2f8f9559d8ea90d2d28, reversing changes made to ed04027d0530048330f973235b519f8990433329.

Dropping collaboration has been paused.  To make sure that messaging does not go out in any future releases I have reverted the commit that added the messaging.


### Test
1. open a note and go to the options menu > Collaborate
2. Confirm that you do not see any messaging about collaboration going away.

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer is required to review these changes, but anyone can perform the review.

### Release
> These changes do not require release notes.
